Public broadcasting of Latvia.js

```diff
@@ -0,0 +1,335 @@
+{
+	"translatorID": "dab3903a-0289-46f8-a74a-eccf7bdb86b6",
+	"label": "Public broadcasting of Latvia",
```

Comment:
You should be more specific here. I would suggest to use here `Latvijas Sabiedriskie Mediji` which also resolves to the acronym of the url `lsm`.